### PR TITLE
Added all Sections - Environment, Stakeholders, Expected Technology, Project Schedule in Tabular Format, Gitworkflow plan and Communication Plan

### PR DESCRIPTION
**Pull request by Kartik1109**
Added all Sections - Envirnoment, Stakeholders, Expected Technology, Project Schedule in Tabular Format

### CODE REVIEW

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 1]
```diff
@@ -240,7 +312,85 @@ \section{Proof of Concept Demonstration Plan}
...
+      \item TensorFlow (TF): The main machine learning framework to train or fine-tune models for tasks such as image classification and segmentation. We may also leverage pre-trained models, like Meta’s Segment Anything Model for segmentation tasks.
+      \item Numpy: A core library for numerical computing in Python, essential for handling arrays and performing mathematical operations efficiently.
+      \item PILLOW: A Python library used for image processing, enabling tasks like image resizing, format conversion, and basic manipulations.
+  \end{itemize}
+  
+  \section{Database:}
+  \begin{itemize}
+      \item SQLAlchemy: A powerful ORM (Object Relational Mapper) for Python that will facilitate database connections and work with both SQL and NoSQL databases.
+      \item SQL (PostgreSQL): Suitable for handling structured data and relationships. Ideal if we require ACID compliance and structured querying.
+      \item NoSQL (MongoDB): More flexible for handling unstructured data, allowing for greater scalability and schema-less design.
```
Comment (KyleJMcMaster) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:375]: I think we should just pick one (SQL/NoSQL) for now. We can always change as the project progresses, but as its written now it seems as though we'll use both

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 2]
```diff
@@ -86,8 +118,48 @@ \section{Team Meeting Plan}
...
+
+5.	Regular Stakeholder Updates: Team members will manage the creation and dissemination of concise updates to stakeholders, keeping them informed on project progress, timelines, and deliverables. \\
+\\
+
+6.	Documentation of Key Communications: All significant external communications and feedback will be documented for future reference, maintaining transparency and accountability across the project.\\
+\\
+
 \wss{Issues on GitHub should be part of your communication plan.}
 
+\section{Git Workflow}
```
Comment (KyleJMcMaster) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:143]: I think this section needs more detail:
image

Seems like we need to cover a more step by step guide, as well as specific descriptions for labels (eg. meeting tag for meetings, documentation tag for docs)

**Review by KyleJMcMaster — CHANGES_REQUESTED** [review]

File: docs/DevelopmentPlan/DevelopmentPlan.tex (hunk 1 above)
Comment (Kartik1109) [reply docs/DevelopmentPlan/DevelopmentPlan.tex]: Yes, I agree. For now I have listed Postgis as our preferred SQL db but we can change in future based on our needs.

File: docs/DevelopmentPlan/DevelopmentPlan.tex (hunk 2 above)
Comment (Kartik1109) [reply docs/DevelopmentPlan/DevelopmentPlan.tex]: I have updated the workflow plan to meet the rubric requirements.

**Review by KyleJMcMaster — APPROVED** [review]
Looks great!

**Comment by Kartik1109** [comment]
> Looks great!

Thanks Kyle :)

**Pull request merged**